scripts/flash-linux.sh

```diff
@@ -24,3 +24,8 @@ if [ -f /etc/init.d/klipper_mcu ]; then
     echo "Attempting host MCU restart..."
     service klipper_mcu restart
 fi
+
+if [ -f /etc/systemd/system/klipper-mcu.service ]; then
+    echo "Attempting host MCU restart..."
+    service klipper-mcu restart
```

Comment:
```suggestion
    systemctl restart klipper-mcu
```
I would use the systemctl command instead of service for systemd services.